tests/functional/artifacts/test_run_results.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from dbt.tests.util import run_dbt
+
+good_model_sql = """
+select 1 as id
+"""
+
+bad_model_sql = """
+something bad
+"""
+
+
+class TestRunResultsTimingSuccess:
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {"model.sql": good_model_sql}
+
+    def test_timing_exists(self, project):
+        results = run_dbt(["run"])
+        assert results.results[0].timing
```

Comment:
`results.results` is a list of `RunResult`s, not `TimingInfo`. `RunResult` has a `timing` attribute on it which is a list of `TimingInfo`s. What I'm doing here is asserting that this list of `TimingInfo`s is not empty.

There should however only ever be one result in `results.results` since only one model is run for each of these tests, so I will add an assertion for that.